index.html

```diff
@@ -7,7 +7,10 @@
 <body>
   <div class="center reamde">
     <h1> Spring 2015!</h1>
-    <a href="david.html">David Zabner's page</a>
+    <ul>
+    	<li><a href="david.html">David Zabner's page</a></li>
```

Comment:
This puts bullet points on the far right side of the page! 
Maybe there is a better way to list these...?

Also the index page is so lacking in style! Could you make it less ugly?